### PR TITLE
copy spreadsheet website from txapu.com

### PR DESCRIPTION
**Pull request by gipsy-king**
will be hosted at http://Civcraft.github.io/RealisticBiomes/spreadsheet/

preview at http://gipsy-king.github.io/RealisticBiomes/spreadsheet/

bonus: anybody who forks this and changes the config can preview his changes in a sane manner at http://USERNAME.github.io/RealisticBiomes/spreadsheet/

### CODE REVIEW

**Comment by WildWeazel** [comment]
Once you open a pull request, any commits to the source branch will automatically be added to the pull.

**Comment by gipsy-king** [comment]
Yea that's what I'm accustomed to from gitlab, but I wasn't seeing the updates here. Maybe because I closed it, my bad.

**Comment by drjawa** [comment]
Should we put this in the civcraftsite repo instead? Want to rationalize our domains so that maybe dev.civcraft.co/ points to all documentation / developer stuff. 

Then again as long as dev.civcraft.co/realisticbiomes points to this then I would be satisfied.

**Comment by WildWeazel** [comment]
sounds good as long as it doesn't interfere with updating the spreadsheet

**Comment by ttk2** [comment]
Most current online documentation is in github wiki's right?

**Comment by drjawa** [comment]
Yes, though this spreadsheet uses some features that can't be done in them.

**Comment by ttk2** [comment]
I would like to prefer to avoid spreading documentation out further.

**Comment by drjawa** [comment]
tldr: Well, currently growth rates are already linked to in a separate and obscure google docs link. 

So linking to `dev.civcraft.co/realisticbiomes` for the growth rate sheet wouldn't increase fragmentation anymore than the current state. 

As far as spreadsheets go though the most github's markdown can do is tables which may not work well nor are they fun to update.

**Comment by ttk2** [comment]
very well then do whats best

**Comment by drjawa** [comment]
For now I'll merge this here for now. Updating wiki shortly.

**Comment by WildWeazel** [comment]
I've been pointing people to this spreadsheet (on txapu) for a while now because I haven't been maintaining mine. The real advantage to this is that it pulls live data from the config so nobody has to update it when we change something, which isn't possible with the repo wiki.

**Comment by drjawa** [comment]
Do you think we should just get rid of this branch then and link to
the txapu page?

**Comment by gipsy-king** [comment]
No, we should linkto http://civcraft.github.io/RealisticBiomes/spreadsheet/ . Now the spreadsheet is under control of the Civcraft org, not of single person. Txapu.com could become unavailable (it's not even my hosting, it belongs to the player txapu0).
